### PR TITLE
Ana/beneficiary is also an Object (NetworkAccount)

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Tested voters, proposers and depositers. Now it is much faster.

Merge! :woman_dancing: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/source/polkadotV0-source.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ class polkadotAPI {
     return api
   }
 
+  async getNetworkAccountInfo(address, api) {
```
Comment (faboweb) [inline api/lib/source/polkadotV0-source.js:40]: nice

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:40]: the identities will get stale. but we shouldn't care about it right now as we release the API frequently

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:40]: how about returning the result after `networkAccountReducer`? that would reduce code

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:40]:
```suggestion
  async getNetworkAccountInfo(address, api, reducers) {
```

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:40]: ah, I see what you mean. And calling `networkAccountReducer` only once inside `getNetworkAccountInfo`.

Good idea

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:40]: yep

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:40]: as a guide: things we store in "store" should already be in the Lunie data model so after reducers.

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:40]: Done

File: api/lib/reducers/polkadotV0-reducers.js [hunk 2]
```diff
@@ -693,6 +693,7 @@ function topVoterReducer(
   validators,
   network
 ) {
+  console.log('\n ACCOUNT INFO', accountInfo)
```
Comment (faboweb) [inline api/lib/reducers/polkadotV0-reducers.js:696]:
```suggestion
```

**Comment by Bitcoinera** [comment]
why it suddenly says there is no changelog?

**Comment by faboweb** [comment]
Can this be merged now?

**Comment by Bitcoinera** [comment]
> Can this be merged now?

Yes

**Comment by Bitcoinera** [comment]
There is changelog. I don't know why the check fails

File: api/lib/source/polkadotV0-source.js [hunk 3]
```diff
@@ -1198,10 +1198,7 @@ class polkadotAPI {
       ),
       topVoters: await Promise.all(
         topVoters.map(async (topVoterAddress) => {
-          const accountInfo = await this.getNetworkAccountInfo(
-            topVoterAddress,
-            api
-          )
+          const accountInfo = await api.derive.accounts.info(topVoterAddress)
```
Comment (faboweb) [inline api/lib/source/polkadotV0-source.js:1201]: why this change?

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:1201]: because validators need to get more data from `identity` like web and so on, all fields that are not present in the `NetworkAccount` Object

Comment (faboweb) [reply api/lib/source/polkadotV0-source.js:1201]: `topVoterReducer` doesn't use these infos. please double check

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:1201]: ah true
